cloud/env/conf.tf

```diff
@@ -0,0 +1,33 @@
+locals {
+  envs = {
+    default = {
+      vast_image_version = "latest"
```

Comment:
Should this use the latest release instead? `latest` here refers to current master. We might also want to consider changing that naming in our CI, it doesn't quite make sense to set it up that way.